Bottom: 4f6b6265d543312f4e8c9d74b6522b4168986bb9
Top:    7af76078e335d5acccea9e83cb4e86eea842ce74
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-26 15:43:36 -0700

Refresh of mpi-finalize

---

diff --git a/src/preload.c b/src/preload.c
index 996d8e0..515cc10 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -804,6 +804,7 @@ int shutdown(int socket, int how)
 {
 	int fd;
 	printf("shutdown how %s\n", how == SHUT_RDWR ? "RDWR" : how == SHUT_RD ? "RD" : "WR");
+	fflush(stdout);
 	return (fd_get(socket, &fd) == fd_rsocket) ?
 		rshutdown(fd, how) : real.shutdown(fd, how);
 }
